### PR TITLE
TOMEE-2824-Translate to Portuguese: examples/jsonb-custom-serializer

### PR DESCRIPTION
**Pull request by Daniel-Dos**
Linkt to Jira:  https://issues.apache.org/jira/browse/TOMEE-2824

### CODE REVIEW

File: examples/jsonb-custom-serializer/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,489 @@
+:index-group: JSON-B
+:jbake-type: page
+:jbake-status: status=published
+= Serializador personalizado JSON-B
+
+Este exemplo mostra como personalizar objetos
```
Comment (hbelmiro) [inline examples/jsonb-custom-serializer/README_pt.adoc:6]:
```suggestion
Este exemplo mostra como personalizar a serialização/desserialização de objetos
```

File: examples/jsonb-custom-serializer/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,489 @@
...
+serialização/desserialização com jsonb para um aplicativo JAX-RS.
```
Comment (hbelmiro) [inline examples/jsonb-custom-serializer/README_pt.adoc:7]:
```suggestion
com jsonb para um aplicativo JAX-RS.
```

File: examples/jsonb-custom-serializer/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,489 @@
...
+
+== Executando e testando o Endpoint
+
+A Aplicação pode ser executado com `mvn clean install tomee:run' se porta
```
Comment (hbelmiro) [inline examples/jsonb-custom-serializer/README_pt.adoc:11]:
```suggestion
A Aplicação pode ser executada com `mvn clean install tomee:run' se a porta
```

File: examples/jsonb-custom-serializer/README_pt.adoc [hunk 4]
```diff
@@ -0,0 +1,489 @@
...
+8080 está disponível, você pode invocar o seguinteendpoint: (GET)
```
Comment (hbelmiro) [inline examples/jsonb-custom-serializer/README_pt.adoc:12]:
```suggestion
8080 está disponível, você pode invocar o seguinte endpoint: (GET)
```

File: examples/jsonb-custom-serializer/README_pt.adoc [hunk 5]
```diff
@@ -0,0 +1,489 @@
...
+    "id": 1, 
+    "name": "name1extra1", 
+    "registration": "Sat Dec 29 19:48:05 CET 2018",
+    "address": null 
+}
+....
+
+== @ApplicationPath
+
+Classe de ponto de entrada JAXRS, da seguinte maneira jaxrs carregará todas as classes e métodos anotados com @Path sem especificá-los.
```
Comment (hbelmiro) [inline examples/jsonb-custom-serializer/README_pt.adoc:60]:
```suggestion
Classe JAXRS de ponto de entrada, da seguinte maneira, o jaxrs carregará todas as classes e métodos anotados com @Path sem especificá-los.
```

File: examples/jsonb-custom-serializer/README_pt.adoc [hunk 6]
```diff
@@ -0,0 +1,489 @@
...
+
+@ApplicationPath("api")
+public class JAXRSApplication extends Application {
+
+}
+....
+
+== Recurso @Path Rest
+
+Classe jaxrs simples com um GET e um POST endpoint
```
Comment (hbelmiro) [inline examples/jsonb-custom-serializer/README_pt.adoc:74]:
```suggestion
Classe jaxrs simples com um endpoint GET e um POST
```

File: examples/jsonb-custom-serializer/README_pt.adoc [hunk 7]
```diff
@@ -0,0 +1,489 @@
...
+    public Jsonb getContext(Class<?> type) {
+        return jsonb;
+    }
+
+}
+....
+
+== Address serializer
+
+Classe serializador de objetos simples que adiciona `modified 'durante a serialização
```
Comment (hbelmiro) [inline examples/jsonb-custom-serializer/README_pt.adoc:148]:
```suggestion
Classe serializadora de objetos simples que adiciona `modified` durante a serialização
```

File: examples/jsonb-custom-serializer/README_pt.adoc [hunk 8]
```diff
@@ -0,0 +1,489 @@
...
+        }
+
+    }
+
+}
+....
+
+== User Deserializer
+
+Create an object from a json
```
Comment (hbelmiro) [inline examples/jsonb-custom-serializer/README_pt.adoc:173]:
```suggestion
Cria um objeto a partir de um json
```

File: examples/jsonb-custom-serializer/README_pt.adoc [hunk 9]
```diff
@@ -0,0 +1,489 @@
...
+INFO - Stopping server services
+Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.31 sec
+
+Results :
+
+Tests run: 2, Failures: 0, Errors: 0, Skipped: 0
+    
+....
+
+=== Inside the jar
```
Comment (hbelmiro) [inline examples/jsonb-custom-serializer/README_pt.adoc:454]:
```suggestion
=== Dentro do jar
```

File: examples/jsonb-custom-serializer/README_pt.adoc [hunk 10]
```diff
@@ -0,0 +1,489 @@
...
+
+javaee-api:8.0 brings in all the dependencies needed to spin up a
```
Comment (hbelmiro) [inline examples/jsonb-custom-serializer/README_pt.adoc:456]:
```suggestion
javaee-api:8.0 traz todas as dependências necessárias para iniciar uma
```

File: examples/jsonb-custom-serializer/README_pt.adoc [hunk 11]
```diff
@@ -0,0 +1,489 @@
...
+working REST application.
```
Comment (hbelmiro) [inline examples/jsonb-custom-serializer/README_pt.adoc:457]: aplicação REST funcionando

File: examples/jsonb-custom-serializer/README_pt.adoc [hunk 12]
```diff
@@ -0,0 +1,489 @@
...
+
+If we look at the jar built by maven, we’ll see the application itself
```
Comment (hbelmiro) [inline examples/jsonb-custom-serializer/README_pt.adoc:459]:
```suggestion
Se olharmos para o jar construído pelo maven, veremos que a aplicação
```

File: examples/jsonb-custom-serializer/README_pt.adoc (hunk 11 above)
Comment (hbelmiro) [inline examples/jsonb-custom-serializer/README_pt.adoc:457]:
```suggestion
aplicação REST funcionando.
```

File: examples/jsonb-custom-serializer/README_pt.adoc [hunk 13]
```diff
@@ -0,0 +1,489 @@
...
+is quite small:
```
Comment (hbelmiro) [inline examples/jsonb-custom-serializer/README_pt.adoc:460]:
```suggestion
é bem pequena:
```

File: examples/jsonb-custom-serializer/README_pt.adoc [hunk 14]
```diff
@@ -0,0 +1,489 @@
...
+  1549 Sat Dec 29 19:10:38 CET 2018 WEB-INF/classes/org/superbiz/JSONBConfiguration.class
+  1241 Sat Dec 29 17:52:48 CET 2018 WEB-INF/web.xml
+     0 Sat Dec 29 19:10:44 CET 2018 META-INF/maven/
+     0 Sat Dec 29 19:10:44 CET 2018 META-INF/maven/org.superbiz/
+     0 Sat Dec 29 19:10:44 CET 2018 META-INF/maven/org.superbiz/jsonb-custom-serializer/
+  1811 Sat Dec 29 17:53:36 CET 2018 META-INF/maven/org.superbiz/jsonb-custom-serializer/pom.xml
+   132 Sat Dec 29 19:10:42 CET 2018 META-INF/maven/org.superbiz/jsonb-custom-serializer/pom.properties
+....
+
+This single jar could be deployed any any compliant Java EE
```
Comment (hbelmiro) [inline examples/jsonb-custom-serializer/README_pt.adoc:487]:
```suggestion
Esse simples jar poderia ser implantado em qualquer implementação Java EE compliant
```

File: examples/jsonb-custom-serializer/README_pt.adoc [hunk 15]
```diff
@@ -0,0 +1,489 @@
...
+implementation. In TomEE you’d simply place it in the
```
Comment (hbelmiro) [inline examples/jsonb-custom-serializer/README_pt.adoc:488]:
```suggestion
. No TomEE você simplesmente colocaria-o no diretório
```

File: examples/jsonb-custom-serializer/README_pt.adoc [hunk 16]
```diff
@@ -0,0 +1,489 @@
...
+`tomee.home/webapps/` directory.
```
Comment (hbelmiro) [inline examples/jsonb-custom-serializer/README_pt.adoc:489]:
```suggestion
 `tomee.home/webapps/` .
```

**Comment by Daniel-Dos** [comment]
thanks @hbelmiro  for review.

**Comment by dblevins** [comment]
Great to see such careful reviews and teamwork!  Nice job @Daniel-Dos  and @hbelmiro

**Pull request merged**